go/minimal/go.mod

```diff
@@ -3,14 +3,13 @@ module github.com/rotationalio/ensign-examples/go/minimal
 go 1.19
 
 require (
-	github.com/oklog/ulid v1.3.1
-	github.com/rotationalio/go-ensign v0.4.3-0.20230321182152-b38bf26f4383
+	github.com/oklog/ulid/v2 v2.1.0
+	github.com/rotationalio/go-ensign v0.4.3-0.20230321190313-00b7d47d1272
```

Comment:
This is the latest as of 31 minutes ago; but it wasn't part of the fix. I'm not sure if using the `ulid/v2` library was part of the fix or not.